portfolio/src/main/data_processing/processor.py

```diff
@@ -0,0 +1,78 @@
+""" Copyright 2019 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. 
+
+processor.py is used to clean csv's and create data suitable to be
+stored within a database."""
+
+from google.cloud import datastore
+import pandas as pd
+import sys
+
+COLUMNS_OF_INTEREST = ['name', 'race', 'date', 'address', 'city',
+                       'state', 'zipcode', 'police budget', 'hospital',
+                       'education', 'cause of death', 'budget',]
+RACE_CATEGORIES = ['American Indian', 'Alaska Native', 'Asian', 'Black',
+                   'African American', 'Native Hawaiian', 'Native American',
+                   'Pacific Islander', 'White', 'Hispanic', 'Latino',]
+TIME_COLUMNS = ['date', 'name']
```

Comment:
nit" add comma after name to be consistent with RACE_CATEGORIES and COLUMNS_OF_INTEREST